pkg/skaffold/server/v3/endpoints.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v3
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/constants"
+	event "github.com/GoogleContainerTools/skaffold/pkg/skaffold/event/v3"
+	proto "github.com/GoogleContainerTools/skaffold/proto/v3"
+)
+
+var (
+	// For Testing
+	resetStateOnBuild  = event.ResetStateOnBuild
+	resetStateOnDeploy = event.ResetStateOnDeploy
+)
+
+func (s *Server) GetState(context.Context, *empty.Empty) (*proto.State, error) {
+	return event.GetState()
+}
+
+func (s *Server) Events(_ *empty.Empty, stream proto.SkaffoldV3Service_EventsServer) error {
+	return event.ForEachEvent(func(event *proto.Event) error {
+		event.Datacontenttype = "text/xml"
```

Comment:
Can you add a comment explaining why `text/xml` is used?  I remember there was some conversation around this during the doc presentation, @briandealwis did you have anything to add here?

Reply:
Fixed. It should be _application/json_. 
Following comment added- 
_grpc gateway converts messages from protobuf format to JSON format, so updating DatacontentType to application/json_

Reply:
This issue is fixed now.